pkgs/top-level/all-packages.nix

```diff
@@ -19004,7 +19004,15 @@ with pkgs;
 
   doclifter = callPackage ../development/tools/misc/doclifter { };
 
-  docutils = with python3Packages; toPythonApplication docutils;
+  docutils = with python3Packages; toPythonApplication (
```

Comment:
Is this temporary?

Reply:
Yep, the intention would be for this to live only until we update `docutils` to at least this version.